rust-src/id/src/account_holder.rs

```diff
@@ -943,11 +951,18 @@ fn compute_pok_sig<
     Ok((prover, secret))
 }
 
+#[derive(SerdeSerialize, SerdeDeserialize)]
```

Comment:
I think seeing that this type is now in the public API, would it make sense to move it to `types.rs`?